src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -87,6 +122,7 @@ public string SelectedLanguage
             {
                 selectedLanguage = value;
                 RaisePropertyChanged(nameof(SelectedLanguage));
+                UpdateSavedChangesLabel();
```

Comment:
This call is repeated too many times. We can add a property change listener so we can listen to changes and call this automatically. That way we can get rid of all these single calls.

Reply:
Something like here: https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs#L636. Single handler can handle all property change I think, then we can filter out properties we need and call `UpdateSavedChangesLabel()` there